main.py

```diff
@@ -4,25 +4,42 @@
 from discord import app_commands
 import json
 import os
+import shutil
 
 from model import Model
 
 try:
+    if not os.path.exists('config.json'):
+        print('No config file detected, creating one...\n')
+        shutil.copy('example_config.json', 'config.json')
+
+        with open('config.json', 'r') as file:
+            data = file.read()
+
+        input_token = input("Paste the token:\n")
+        new_data = data.replace('Your_token_here', input_token)
+
+        with open('config.json', 'w') as file:
+            file.write(new_data)
+
+        file.close()
+
     with open('config.json') as f:
         config = json.load(f)
+
     GUILD_OBJ = discord.Object(id=config['guild_id'])
     DEFAULT_NEGATIVE_SFW = config['default_sfw_negative_prompt']
     DEFAULT_NEGATIVE_NSFW = config['default_nsfw_negative_prompt']
     EXAMPLE = config['example']
     DEFAULT_HEIGHT = config['default_height']
     DEFAULT_WIDTH = config['default_width']
     client = commands.Bot(intents=discord.Intents.all(), command_prefix=config['command_prefix'])
+
 except KeyError as e:
     print(f'Cant find key value! Update your config file!\n{e}')
     exit(1)
-except (FileNotFoundError, OSError) as e:
-    print(f'Cant open a config file!\n{e}')
-    exit(1)
+# except (FileNotFoundError, OSError) as e:
+#     print(f'Cant open a config file.\n{e}')
```

Comment:
```suggestion
 except (FileNotFoundError, OSError) as e:
    print(f'Cant open a config file.\n{e}')
    exit(1)
```
if something goes wrong